src/main/java/org/apache/commons/compress/archivers/dump/DumpArchiveConstants.java

```diff
@@ -26,7 +26,7 @@ public final class DumpArchiveConstants {
      * The type of compression.
      */
     public enum COMPRESSION_TYPE {
-        ZLIB(0), BZLIB(1), LZO(2);
+        UNKNOWN(-1), ZLIB(0), BZLIB(1), LZO(2);
```

Comment:
Is this required? Does it avoid an NPE? I do not see it used outside of this enum,